packages/scanner-global-library/src/navigation-hooks.ts

```diff
@@ -15,7 +15,10 @@ export class NavigationHooks {
         @inject(PageConfigurator) public readonly pageConfigurator: PageConfigurator,
         @inject(PageResponseProcessor) protected readonly pageResponseProcessor: PageResponseProcessor,
         @inject(PageHandler) protected readonly pageRenderingHandler: PageHandler,
+        private readonly scrollTimeoutMsecs = 15000,
```

Comment:
we can increase both `scrollTimeoutMsecs` and `pageRenderingTimeoutMsecs` to 30 secs and make them equal